docs/generated/galasactl_runs_get.md

```diff
@@ -16,6 +16,7 @@ galasactl runs get [flags]
       --active             parameter to retrieve runs that have not finished yet. Cannot be used in conjunction with --name or --result flag.
       --age string         the age of the test run(s) we want information about. Supported formats are: 'FROM' or 'FROM:TO', where FROM and TO are each ages, made up of an integer and a time-unit qualifier. Supported time-units are 'w' (weeks), 'd' (days), 'h' (hours), 'm' (minutes). If missing, the TO part is defaulted to '0h'. Examples: '--age 1d', '--age 6h:1h' (list test runs which happened from 6 hours ago to 1 hour ago). The TO part must be a smaller time-span than the FROM part.
       --format string      output format for the data returned. Supported formats are: 'details', 'raw', 'summary'. (default "summary")
+      --group string       the name of the group to return tests under that group. Cannot be used in conjunction with --requestor, --result or --active flags
```

Comment:
Do we have any unit tests that pass `--group` in alongside `--requestor`, `--result` or `--active`?